tox.ini

```diff
@@ -39,6 +47,7 @@ allowlist_externals =
 python =
     3.8: py38, pytest-min
     3.9: py39
+    3.10.8: py3108
```

Comment:
Hmm...something's not working with this mapping. The Python 3.10.8 runner uses the `py310` environment, not the `py3108` environment.

It does work locally with pyenv and [tox-pyenv-redux](https://github.com/un-def/tox-pyenv-redux), at least.

Reply:
Python 3.12 should also emit deprecation warnings, albeit less aggressively. If we cannot reproduce the errors with 3.12, it's fine for me to not have regression tests for the changes.

Reply:
Removed the `p3108` regression test attempts.